Specs/NumberWithUnit/German/DimensionModel.json

```diff
@@ -363,5 +363,41 @@
     "Input": "2:00 pm",
     "NotSupported": "java",
     "Results": []
+  },
+  {
+    "Input": "12 Pfund Tasse Kaffee",
```

Comment:
Can you also add a case for "12 - Pfund"?

Reply:
added